componentLibraries/defaultLibrary/Hydraulic/Volumes&Lines/HydraulicAckumulator.hpp

```diff
@@ -269,7 +279,8 @@ xmp,0.1,1)*Power(V0*limit(1 - xmp,0.1,1),-1 - kappa));
           jacobianMatrix[3][4] = 0;
           jacobianMatrix[4][0] = 0;
           jacobianMatrix[4][1] = 0;
-          jacobianMatrix[4][2] = -(Zc1*onPositive(p1));
+          jacobianMatrix[4][2] = -(Zc1*dxLowLimit(c1 + \
```

Comment:
It would be easier on the eyes of the line breaks did not occur, or at least allow longer lines, I think 160 characters, I dont remember if we decided what the maximum should be in Hopsan code.

I realize that the generator would need to be rewritten, so not super important in this PR